File: Livro - Módulo Java/4 - Estrutura de Repetição/Exercicio23.java
import java.util.Scanner;

public class Exercicio23 {
    public static void main(String[] args) {
        // Faça um programa que calcule a soma dos números pares e ímpares
        // a partir de um valor inicial e final informados pelos usuários.

        Scanner scanner = new Scanner(System.in);

        System.out.print("Informe o valor inicial: ");
        int valorInicial = scanner.nextInt();

        System.out.print("Informe o valor final: ");
        int valorFinal = scanner.nextInt();

        calcularSoma(valorInicial, valorFinal);
    }

    public static void calcularSoma(int valorInicial, int valorFinal) {
        if (valorInicial > valorFinal) {
            System.out.println("Intervalo inválido. O valor inicial deve ser menor ou igual ao valor final.");
            return;
        }

        int somaPares = 0;
        int somaImpares = 0;

        for (int i = valorInicial; i <= valorFinal; i++) {
            if (i % 2 == 0) {
                somaPares += i;
            } else {
                somaImpares += i;
            }
        }

        System.out.println("Soma dos números pares: " + somaPares);
        System.out.println("Soma dos números ímpares: " + somaImpares);
    }
}

File: Livro - Módulo Python/4 - Estrutura de Repetição/Exercicio.23.py
#23. Faça um programa que calcule a soma dos números pares e impares a partir de um valor inicial e final 
# informados pelos usuários. Por exemplo, se usuário informou 120 e 130, o programa deverá calcular a soma
# dos números pares : 120 + 122 + 124 + 126 +128 + 130, além dos números impares: 121 + 123 + 125 + 127 + 129.

valor_inicial = int(input("Digite o valor inicial: "))
valor_final = int(input("Digite o valor final: "))

somaPares = 0
somaImpares = 0

for numero in range(valor_inicial, valor_final + 1):
    if numero % 2 == 0:  
        somaPares += numero
    else:
        somaImpares += numero

print("Soma dos números pares:", somaPares)
print("Soma dos números ímpares:", somaImpares)